m4/ax_cxx_compile_stdcxx_17.m4

```diff
@@ -0,0 +1,35 @@
+# =============================================================================
```

Comment:
nowadays one can get these from the `autoconf-archive` package instead of pulling in your own, but perhaps instead of either we can let cmake win and ditch autotools completely at this point. i personally like autotools but the rest of the world is seemingly all in on cmake (which does have its niceties)